packages/artillery/lib/cli/common-flags.js

```diff
@@ -23,7 +23,8 @@ const CommonRunFlags = {
     description: 'Write a JSON report to file'
   }),
   dotenv: Flags.string({
-    description: 'Path to a dotenv file to load environment variables from'
+    description:
+      'Path to a dotenv (.env) file to load environment variables from'
```

Comment:
The `.env` is redundant here. (and those files don't need to be called `.env`)

Reply:
Yes I know. I was basing it off the fact that some of the flag descriptions wrote .env and others dotenv. I'll just remove it .env then 👍🏼